test/e2e/sched/sched_test.go

```diff
@@ -174,5 +181,60 @@ var _ = Describe("[Scheduler] imageReplacement", func() {
 				return true
 			}).WithTimeout(time.Minute * 2).WithPolling(time.Second * 30).Should(BeTrue())
 		})
+		It("should reflect changes in cacheResyncPeriod when configured", func() {
+			nroSchedObj := objects.TestNROScheduler()
+			Expect(e2eclient.Client.Get(context.TODO(), client.ObjectKeyFromObject(nroSchedObj), nroSchedObj)).ToNot(HaveOccurred())
+
+			t := time.Minute * 5
```

Comment:
let's double the value got from previous spec to prevent the (unlikely) case we change the default value to 5 minutes.